### PR TITLE
kwrite: add page

### PR DESCRIPTION
**Pull request by RedHeadphone**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
kwrite 21.04.2

### CODE REVIEW

File: pages/linux/kwrite.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Open multiple text files:
+
+`kwrite {{file1 file2 ...}}`
+
+- Open a text file with a specific encoding:
+
+`kwrite --encoding={{UTF-8}} {{path/to/file}}`
+
+- Navigate to a specific line and column in a text document.:
```
Comment (nicokosi) [inline pages/linux/kwrite.md:18]:
```suggestion
- Navigate to a specific line and column in a text document:
```

Comment (nicokosi) [inline pages/linux/kwrite.md:18]: I suggest using the same wording ("text file"):
```suggestion
- Navigate to a specific line and column in a text file:
```

**Review by blueskyson — COMMENTED** [review]
Look good to me. Thanks for contribution.

**Comment by marchersimon** [comment]
@blueskyson you didn't actually approve 🤔

File: pages/linux/kwrite.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Open multiple text files:
+
+`kwrite {{file1 file2 ...}}`
+
+- Open a text file with a specific encoding:
+
+`kwrite --encoding={{UTF-8}} {{path/to/file}}`
+
+- Navigate to a specific line and column in a text file:
```
Comment (marchersimon) [inline pages/linux/kwrite.md:18]:
```suggestion
- Open a text file and navigate to a specific line and column:
```

File: pages/linux/kwrite.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# kwrite
+
+> Text editor of the KDE Desktop project.
```
Comment (marchersimon) [inline pages/linux/kwrite.md:3]: What do you think about this?
```suggestion
> Text editor of the KDE Desktop project.
> See also `kate`.
```

Comment (RedHeadphone) [reply pages/linux/kwrite.md:3]: I haven't used kate. Its not pre installed in kde neon so never used it. Do I add it ?

Comment (marchersimon) [reply pages/linux/kwrite.md:3]: Not sure which version of KDE you're using, but on Manjaro KDE, Kate is preinstalled and Kwrite isn't. But I think we should add it.

Comment (RedHeadphone) [reply pages/linux/kwrite.md:3]: image

Comment (RedHeadphone) [reply pages/linux/kwrite.md:3]: ok i will add that line

**Review by marchersimon — APPROVED** [review]
Thanks @RedHeadphone

File: pages/linux/kwrite.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`kwrite {{file1 file2 ...}}`
+
+- Open a text file with a specific encoding:
+
+`kwrite --encoding={{UTF-8}} {{path/to/file}}`
+
+- Open a text file and navigate to a specific line and column:
+
+`kwrite -l {{line_number}} -c {{column_number}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/linux/kwrite.md:21]: We prefer long arguments if available
```suggestion
`kwrite --line {{line_number}} --column {{column_number}} {{path/to/file}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**